components/infobox/commons/infobox_skill.lua

```diff
@@ -0,0 +1,80 @@
+local BasicInfobox = require('Module:Infobox/Basic')
+local Class = require('Module:Class')
+local Cell = require('Module:Infobox/Cell')
+local Namespace = require('Module:Namespace')
+local String = require('Module:StringUtils')
+local Hotkey = require('Module:Hotkey')
+
+local Skill = Class.new(BasicInfobox)
+
+function Skill.run(frame)
+	local skill = Skill(frame)
+	return skill:createInfobox(frame)
+end
+
+function Skill:createInfobox(frame)
+	local infobox = self.infobox
+	local args = self.args
+
+	if String.isEmpty(args.informationType) then
+		error('You need to specify an informationType')
```

Comment:
Perhaps give a suggestion? Might make the error easier to grasp, `informationType` seems a little abstract on its own.

Reply:
please check if is good now^^